lib/async_hooks.js

```diff
@@ -61,13 +61,7 @@ async_wrap.setupHooks({ init,
 
 // Used to fatally abort the process if a callback throws.
 function fatalError(e) {
-  if (typeof e.stack === 'string') {
-    process._rawDebug(e.stack);
-  } else {
-    const o = { message: e };
-    Error.captureStackTrace(o, fatalError);
-    process._rawDebug(o.stack);
-  }
+  process._rawDebug(e.stack);
```

Comment:
Nevermind re https://github.com/nodejs/node/pull/14051#discussion_r125295530, I'll move it to `test/common`